configurations/mt-staging.ts

```diff
@@ -115,6 +115,209 @@ const preEngagementConfig: PreEngagementConfig = {
           required: true,
         }
     },
+    {
+      label: 'Select your Age/Età/літа',
+      type: 'SelectItem',
+      attributes: {
+        name: 'age',
+        required: true,
+        readOnly: false,
+      },
+      options: [
+        {
+          value: "",
+          label: ""
+        },
+        {
+          value: "Unborn",
+          label: "Unborn"
+        },
+        {
+          value: "1",
```

Comment:
I think you might to add leading zeros here, to match what is in the [forms ](https://github.com/techmatters/flex-plugins/blob/master/hrm-form-definitions/form-definitions/mt/v1/tabbedForms/ChildInformationTab.json). You need to add the "00" option to make it the same as in the form and we don't have an unborn option there either.